packages/server/src/seed/seed.controller.ts

```diff
@@ -35,7 +37,7 @@ export class SeedController {
   constructor(
     private connection: Connection,
     private seedService: SeedService,
-  ) { }
+  ) {}
 
   @Get('delete')
   async deleteAll(): Promise<string> {
```

Comment:
Don't we need to delete from OrganizationCourseModel and OrganizationUserModel also? Because of FK constraints.